### PR TITLE
add aws-ipi-disconnected-cco-manual-sts-ep coverage

### PR DESCRIPTION
**Pull request by jianlinliu**
1. add aws-ipi-disconnected-cco-manual-sts-ep ci coverage
2. recover `reporter_config` mentioned in https://github.com/openshift/release/pull/35062/files#r1063213459

### CODE REVIEW

**Comment by jianlinliu** [comment]
cc @yunjiang29 and @lxia for review

**Comment by yunjiang29** [comment]
/lgtm

@jianlinliu thanks for adding these jobs.

**Comment by jianlinliu** [comment]
@LiangquanLi930 @sunzhaohua2 pls help review `reporter_config` part.

**Comment by liangxia** [comment]
@kuiwang02 could you help review the test filter for ginkgo test ?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-stable.yaml [hunk 1]
```diff
@@ -163,6 +163,42 @@ tests:
...
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: m6g.xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
+      E2E_RUN_TAGS: '@arm64 and @aws-ipi and @disconnected and @network-ovnkubernetes
+        and not @fips and @destructive'
+      OCP_ARCH: arm64
+      TAG_VERSION: '@4.12'
+      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~ConnectedOnly&;~HyperShiftMGMT&
```
Comment (kuiwang02) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-stable.yaml:198]: @jianlinliu it should be `~ChkUpgrade&;~NonPreRelease&;Serial;Disruptive;~ConnectedOnly&;~HyperShiftMGMT&`

**Comment by kuiwang02** [comment]
> @kuiwang02 could you help review the test filter for ginkgo test ?

Thanks. it is done with one comment.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-stable.yaml (hunk 1 above)
Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-stable.yaml]: Good catch, updated.

**Pull request merged**